cliolog/new.go

```diff
@@ -49,3 +67,43 @@ func WithNoColor(noColor *bool) func(*Options) {
 		o.NoColor = noColor
 	}
 }
+
+type FileLoggerConfig struct {
+	Filename       string
+	MaxSize        *int
+	MaxBackups     *int
+	MaxAge         *int
+	ShouldCompress *bool
```

Comment:
minor nitpick: don't make these pointers, it makes the config more annoying to use. Just add documentation for each value and note that defaults values will be used if they aren't provided:
```suggestion
        // MaxSize is the maximum size of the log file in MB.
        // If set to 0 the default max size is used (1MB)
	MaxSize        int
	// MaxBackups is .....
        // If set to 0 the default ...
	MaxBackups     int
	// MaxAge is .....
        // If set to 0 the default ...
	MaxAge         int
	// ShouldCompress is .....
	ShouldCompress bool
```